### PR TITLE
Pytesseract

### PR DESCRIPTION
**Pull request by adbeda**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pytesseract/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+source:
+  url: https://github.com/madmaze/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: b389cf1208df8567f953b8da5b716b28f4c42f149f367b4e51b73ef651a55707
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (scopatz) [inline recipes/pytesseract/meta.yaml]: These should be in host, not build

File: recipes/pytesseract/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - python
+    - pip
+
+  run:
+    - python
+    - flake8
+    - pep8-naming
+    - flake8-quotes
+    - pytest
+    - pillow 5.4.1
```
Comment (scopatz) [inline recipes/pytesseract/meta.yaml]: Does this pinning need to be this exact?

File: recipes/pytesseract/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/madmaze/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: b389cf1208df8567f953b8da5b716b28f4c42f149f367b4e51b73ef651a55707
+
+build:
+  number: 0
```
Comment (scopatz) [inline recipes/pytesseract/meta.yaml]: Can this be `noarch: python`?

File: recipes/pytesseract/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (scopatz) [inline recipes/pytesseract/meta.yaml]: Can you use the install line from the example, https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

```yaml
script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/pytesseract/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+   as it can read all image types supported by the Pillow and Leptonica
+   imaging libraries, including jpeg, png, gif, bmp, tiff, and others.
+   Additionally, if used as a script, Python-tesseract will print the
+   recognized text instead of writing it to a file
+
+
+extra:
+  recipe-maintainers:
+    - adbeda
+    - agcopenhaver
```
Comment (scopatz) [inline recipes/pytesseract/meta.yaml]: @agcopenhaver are you OK with being a maintainer?

Comment (agcopenhaver) [reply recipes/pytesseract/meta.yaml]: Sure thing @scopatz

File: recipes/pytesseract/meta.yaml (hunk 4 above)
Comment (adbeda) [reply recipes/pytesseract/meta.yaml]: @scopatz I was looking at [click](https://github.com/conda-forge/conda-smithy/blob/7ee8fb785642d2d5ce5244aa7db2d9912bbc45f0/tests/recipes/click-test-feedstock/recipe/meta.yaml#L19) and [pytest-mock](https://github.com/conda-forge/pytest-mock-feedstock/blob/c99e9c7add9dc7ae0d7445a1b097d235e14ee9ea/recipe/meta.yaml#L17)

**Comment by adbeda** [comment]
@scopatz `noarch: python` specification failed for windows.

**Comment by chrisburr** [comment]
@adbeda It fails because you need `entry_points`, see:
* https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points
* https://github.com/madmaze/pytesseract/blob/master/setup.py#L34
You can then make this `noarch`

**Comment by adbeda** [comment]
@chrisburr got it. thanks!

**Pull request merged**